.github/workflows/test.yml

```diff
@@ -10,8 +10,6 @@ on:
     branches:
       - main
   pull_request:
-  # manual trigger
-  workflow_dispatch:
```

Comment:
qq: will this prevent our ability to manually re-run failed tests?

Reply:
no, it will just prevent running tests on arbitrary commits outside of PRs, as through https://github.com/Uniswap/interface/actions/workflows/test.yml.